.github/workflows/test-process_extended.yml

```
@@ -69,6 +69,9 @@ jobs:
         with:
           chrome-version: 119.0.6045.105
 
+      - uses: Install ImageMagick
+        run: sudo apt install imagemagick
```

please, change in all files
```suggestion
      - name: Install ImageMagick
        run: sudo apt install imagemagick
```